tbi.datacommons.io/portal/gitops.json

```diff
@@ -117,11 +117,11 @@
       ]
     },
     "login": {
-      "title": "BRAIN Commons",
+      "title": "TBI Commons",
       "subTitle": "Explore, Analyze, and Share Data",
-      "text": "The BRAIN Commons is a scalable cloud-based platform for computational discovery designed for the brain health community. The BRAIN Commons empowers the global research community by providing access to multi-modal data, state-of-the-art analytic tools and a secure interoperable system for data sharing to accelerate discovery and development of therapies, diagnostic tests, and other technologies for the treatment and prevention of diseases impacting the brain.",
+      "text": "The TBI Commons is a scalable cloud-based platform for computational discovery designed for the brain health community. The BRAIN Commons empowers the global research community by providing access to multi-modal data, state-of-the-art analytic tools and a secure interoperable system for data sharing to accelerate discovery and development of therapies, diagnostic tests, and other technologies for the treatment and prevention of diseases impacting the brain.",
```

Comment:
there are still some words saying `BRAIN Commons` in here and for some other buttons, do we want to change them all?

Reply:
i think i got them all now